src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -85,6 +85,22 @@ azure() {
         ${optionals}
 }
 
+gcp() {
+test_vars GCP_CREDENTIALS
+
+[[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-gcp-image-name ${PODVM_IMAGE_NAME} "
+[[ "${GCP_PROJECT_ID}" ]] && optionals+="-gcp-project-id ${GCP_PROJECT_ID} "
+[[ "${GCP_ZONE}" ]] && optionals+="-gcp-zone ${GCP_ZONE} " # if not set retrieved from IMDS
+[[ "${GCP_MACHINE_TYPE}" ]] && optionals+="-gcp-machine-type ${GCP_MACHINE_TYPE} " # default e2-medium
+[[ "${GCP_NETWORK}" ]] && optionals+="-gcp-network ${GCP_NETWORK} " # defaults to 'default'
+
+set -x
+exec cloud-api-adaptor gcp \
+	-pods-dir /run/peerpod/pods \
+	${optionals} \
+	-socket /run/peerpod/hypervisor.sock
```

Comment:
I think -socket isn't needed because it defaults to /run/peerpod/hypervisor.sock already.

Reply:
I was just playing safe, since others are using it as well. But I will remove with the next force-push.